src/code/mtxuty-cvt.c

```diff
@@ -7,8 +7,8 @@ void MtxConv_F2L(Mtx* m1, MtxF* m2) {
     LogUtils_CheckNullPointer("m1", m1, "../mtxuty-cvt.c", 31);
     LogUtils_CheckNullPointer("m2", m2, "../mtxuty-cvt.c", 32);
 
-    for (i = 0; i < 4; i++) {
-        for (j = 0; j < 4; j++) {
+    for (i = 0; i < ARRAY_COUNT(m2->mf); i++) {
+        for (j = 0; j < ARRAY_COUNT(m2->mf[i]); j++) {
```

Comment:
Matrices in oot and in graphics rendering in general are 4x4 and anyone trying to change this is doing very *very* wild things

using array_count for this would make it confusing and harm readability more than anything